client/src/main.rs

```diff
@@ -48,15 +46,8 @@ async fn main() -> anyhow::Result<()> {
             buyer_ecash_pubkey = get_user_input("Enter buyer's ecash pubkey: ").await?;
             String::from("seller")
         }
-        "3" => {
-            nostr_client = NostrClient::new(&env::var("ESCROW_NSEC")?).await?;
-            //println!("Coordinator npub: {}", nostr_client.get_npub().await?);
-            let mut escrow_provider = EscrowProvider::setup(nostr_client, ecash_wallet).await?;
-            escrow_provider.run().await?;
-            return Ok(());
-        }
         _ => {
```

Comment:
I guess looping back to user input, panicing or exiting with an error message would be appropiate here if the user does not select the correct trading mode. But this function is bit cluttered anyway, maybe we can split it up a bit later.

```suggestion
        _ => {
            panic!("Wrong trading mode selected. Select either (1) buyer or (2) seller");
```

Reply:
Yes, I agree.